lantern/flask/flask_login_and_templates/practice/grocery_store/routes/main.py

```diff
@@ -15,10 +15,40 @@ def index():
 @login_required
 def profile():
     return render_template(
-        "profile.html", user=current_user.name, email=current_user.email
+        "profile.html", user=current_user.name, email=current_user.email,
     )
 
 
+@main.route("/user_orders_list")
+@login_required
+def orders():
+    user_orders_list = []
+    for order in current_user.orders:
+        data = {
+            "store": order.store.name,
+            "date": order.created_time,
+            "price": sum([good.good.price for good in order.order_lines]),
+            "goods": {good.good.name: good.good.price for good in order.order_lines},
+        }
+        user_orders_list.append(data)
+    return render_template("orders.html", orders=user_orders_list)
+
+
+@main.route("/manage_stores")
+@login_required
+def stores():
+    user_managed_stores = []
+    for store in current_user.manage_stores:
+        data = {
+            "store_id": store.store_id,
+            "name": store.name,
+            "city": store.city,
+            "address": store.address,
+        }
+        user_managed_stores.append(data)
```

Comment:
actually you do not need this loop, you may just return current_user.manage_stores and it will be the same

Reply:
@dimigor, done, thanks:)